db/schema.rb

```diff
@@ -667,6 +1544,34 @@
+  add_foreign_key "datasources_statistic_research_questions", "datasources_statistic", column: "to_statistic_id", name: "datasources_statisti_to_statistic_id_f4b3621d_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statistic_users", "auth_user", column: "user_id", name: "datasources_statistic_Users_user_id_c2750f30_fk_auth_user_id", deferrable: :deferred
+  add_foreign_key "datasources_statistic_users", "datasources_statistic", column: "statistic_id", name: "datasources_statisti_statistic_id_ddfaefdc_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statistic_visible_by", "auth_user", column: "user_id", name: "datasources_statisti_user_id_f06a25ab_fk_auth_user", deferrable: :deferred
+  add_foreign_key "datasources_statistic_visible_by", "datasources_statistic", column: "statistic_id", name: "datasources_statisti_statistic_id_8e74144b_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statisticattribute", "datasources_statistic", column: "statistic_id", name: "datasources_statisti_statistic_id_dd17ea57_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statisticrefresh", "auth_user", column: "creator_id", name: "datasources_statisti_creator_id_986147ac_fk_auth_user", deferrable: :deferred
+  add_foreign_key "datasources_statisticrefresh", "datasources_datasource", column: "data_source_id", name: "datasources_statisti_data_source_id_40d7dc16_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statisticrefresh", "datasources_statistic", column: "statistic_id", name: "datasources_statisti_statistic_id_2cfb46c7_fk_datasourc", deferrable: :deferred
+  add_foreign_key "datasources_statisticrefresh", "django_celery_beat_periodictask", column: "task_source_id", name: "datasources_statisti_task_source_id_0f2577a9_fk_django_ce", deferrable: :deferred
+  add_foreign_key "django_admin_log", "auth_user", column: "user_id", name: "django_admin_log_user_id_c564eba6_fk_auth_user_id", deferrable: :deferred
+  add_foreign_key "django_admin_log", "django_content_type", column: "content_type_id", name: "django_admin_log_content_type_id_c4bce8eb_fk_django_co", deferrable: :deferred
+  add_foreign_key "django_celery_beat_periodictask", "django_celery_beat_clockedschedule", column: "clocked_id", name: "django_celery_beat_p_clocked_id_47a69f82_fk_django_ce", deferrable: :deferred
+  add_foreign_key "django_celery_beat_periodictask", "django_celery_beat_crontabschedule", column: "crontab_id", name: "django_celery_beat_p_crontab_id_d3cba168_fk_django_ce", deferrable: :deferred
+  add_foreign_key "django_celery_beat_periodictask", "django_celery_beat_intervalschedule", column: "interval_id", name: "django_celery_beat_p_interval_id_a8ca27da_fk_django_ce", deferrable: :deferred
+  add_foreign_key "django_celery_beat_periodictask", "django_celery_beat_solarschedule", column: "solar_id", name: "django_celery_beat_p_solar_id_a87ce72c_fk_django_ce", deferrable: :deferred
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹